cyral/data_source_cyral_integration_idp_saml.go

```diff
@@ -161,6 +174,40 @@ func dataSourceIntegrationIdPSAML() *schema.Resource {
 										Type:        schema.TypeString,
 										Computed:    true,
 									},
+									"url": {
+										Description: "URL where the metadata document can be downloaded.",
+										Type:        schema.TypeString,
+										Computed:    true,
+									},
+									"entity_id": {
+										Description: "Entity ID defined in th SAML Metadata XML.",
+										Type:        schema.TypeString,
+										Computed:    true,
+									},
+									"single_logout_url": {
+										Description: "The single logout URL defined in the SAML Metadata XML (SL0).",
+										Type:        schema.TypeString,
+										Computed:    true,
+									},
+									"assertion_consumer_services": {
+										Description: "The Assertion Consumer Services defined in the SAML Metadata XML.",
+										Type:        schema.TypeSet,
```

Comment:
See my comment about that below.
```suggestion
										Type:        schema.TypeList,
```

Reply:
addr [637f75e](https://github.com/cyralinc/terraform-provider-cyral/pull/394/commits/637f75e408adf92993fa7023e968c975b2bfc972)